HDFEventOutputer.h

```diff
@@ -0,0 +1,58 @@
+#if !defined(HDFEventOutputer_h)
+#define HDFEventOutputer_h
+
+
+#include <vector>
+#include <string>
+#include <cstdint>
+#include <fstream>
+
+
+#include "OutputerBase.h"
+#include "EventIdentifier.h"
+#include "SerializeStrategy.h"
+#include "DataProductRetriever.h"
+#include "pds_writer.h"
+
+#include "SerialTaskQueue.h"
+
+#include "HDFCxx.h"
+
+using product_t = std::vector<char>;
```

Comment:
Please do not place a `using` in a header in the global namespace. In fact, it isn't even used in the header so should be removed.
```suggestion
```